run freezing in nightly

eellison opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #106146
* #106098
* #106092
* __->__ #106097

We could switch to every other day or something but inference is cheaper than training anyway.

eellison committed:
run freezing in nightly

[ghstack-poisoned]

desertfire reviewed (commented):
If you meant to have it run nightly, you need to something similar to https://github.com/pytorch/pytorch/pull/106077/

eellison committed:
Update on "run freezing in nightly"


We could switch to every other day or something but inference is cheaper than training anyway.


[ghstack-poisoned]

eellison commented:
@pytorchbot merge

eellison commented:
@pytorchbot merge -f "dont need full test suite for change in nightly config"

eellison committed:
Update on "run freezing in nightly"


We could switch to every other day or something but inference is cheaper than training anyway.


[ghstack-poisoned]

eellison commented:
@pytorchbot merge
